storage/embedded-tools/storage-migrator/src/main/resources/META-INF/mappings/package.mappings

```diff
@@ -0,0 +1,99 @@
+one.microstream.afs.azure.storage.types=org.eclipse.store.afs.azure.storage.types
+one.microstream.afs.blobstore.types=org.eclipse.store.afs.blobstore.types
+one.microstream.afs.googlecloud.firestore.types=org.eclipse.store.afs.googlecloud.firestore.types
+one.microstream.afs.hazelcast.types=org.eclipse.store.afs.hazelcast.types
+one.microstream.afs.kafka.types=org.eclipse.store.afs.kafka.types
+one.microstream.afs.nio.types=org.eclipse.store.afs.nio.types
+one.microstream.afs.oracle.coherence.types=org.eclipse.store.afs.oracle.coherence.types
+one.microstream.afs.oraclecloud.objectstorage.types=org.eclipse.store.afs.oraclecloud.objectstorage.types
+one.microstream.afs.redis.types=org.eclipse.store.afs.redis.types
+one.microstream.afs.sql.types=org.eclipse.store.afs.sql.types
+one.microstream.branching=org.eclipse.serializer.branching
+one.microstream.bytes=org.eclipse.serializer.bytes
+one.microstream.chars=org.eclipse.serializer.chars
+one.microstream.collections=org.eclipse.serializer.collections
+one.microstream.collections.interfaces=org.eclipse.serializer.collections.interfaces
+one.microstream.collections.lazy=org.eclipse.serializer.collections.lazy
+one.microstream.collections.old=org.eclipse.serializer.collections.old
+one.microstream.collections.sorting=org.eclipse.serializer.collections.sorting
+one.microstream.collections.types=org.eclipse.serializer.collections.types
+one.microstream.concurrency=org.eclipse.serializer.concurrency
+one.microstream.entity=org.eclipse.serializer.entity
+one.microstream.equality=org.eclipse.serializer.equality
+one.microstream.exceptions=org.eclipse.serializer.exceptions
+one.microstream.functional=org.eclipse.serializer.functional
+one.microstream.hashing=org.eclipse.serializer.hashing
+one.microstream.io=org.eclipse.serializer.io
+one.microstream.math=org.eclipse.serializer.math
+one.microstream.memory=org.eclipse.serializer.memory
+one.microstream.memory.android=org.eclipse.serializer.memory.android
+one.microstream.memory.sun=org.eclipse.serializer.memory.sun
+one.microstream.meta=org.eclipse.serializer.meta
+one.microstream.reference=org.eclipse.serializer.reference
+one.microstream.reflect=org.eclipse.serializer.reflect
+one.microstream.time=org.eclipse.serializer.time
+one.microstream.typing=org.eclipse.serializer.typing
+one.microstream.util=org.eclipse.serializer.util
+one.microstream.util.cql=org.eclipse.serializer.util.cql
+one.microstream.util.iterables=org.eclipse.serializer.util.iterables
+one.microstream.util.logging=org.eclipse.serializer.util.logging
+one.microstream.util.similarity=org.eclipse.serializer.util.similarity
+one.microstream.util.traversing=org.eclipse.serializer.util.traversing
+one.microstream.util.xcsv=org.eclipse.serializer.util.xcsv
+one.microstream.wrapping=org.eclipse.serializer.wrapping
+one.microstream.configuration.exceptions=org.eclipse.store.configuration.exceptions
+one.microstream.configuration.types=org.eclipse.store.configuration.types
```

Comment:
Now I don't know, wasn't this moved to the serializer?

Reply:
Yes, we moved it.